src/static/riot/competitions/detail/_tabs.tag

```diff
@@ -217,12 +217,13 @@
                 <loader></loader>
             </div>
             <!-- Tab Content !-->
-            <div show="{!loading && !_.isEmpty(competition.leaderboards)}">
+            <div show="{!loading && !_.isEmpty(leaderboard_phases)}">
                 <div class="ui button-container">
-                    <div class="ui inline button {active: selected_leaderboard_index == leaderboard.id}"
-                         each="{ leaderboard in competition.leaderboards }"
-                         onclick="{ leaderboard_selected.bind(this, leaderboard) }">{ leaderboard.title }
+                    <div class="ui inline button {active: selected_phase_index == phase.id}"
+                         each="{ phase in leaderboard_phases }"
+                         onclick="{ phase_selected.bind(this, phase) }">{ phase.name }
                     </div>
+                </div>
```

Comment:
For some reason, the download button next to this isn't inline any more.
![image](https://user-images.githubusercontent.com/16579931/92074912-2880c500-ed6c-11ea-883e-f4b99130bc78.png)